package.json

```diff
@@ -173,5 +173,9 @@
   "yoshi": {
     "exports": "wix-ui-tpa",
     "tpaStyle": true
+  },
+  "prettier": {
+    "singleQuote": true,
+    "trailingComma": "all"
```

Comment:
I think we already get these from yoshi, no?

Reply:
Perhaps, but it seems that Husky runs with different defaults (or doesn't take from Yoshi) - which probably leads to the conflicts...

For VSCode I can say this fixes the problem, but for WebStrom it might be needed for each contributor to configure manually (not sure if Prettier plugin is already supported well in WebStorm?).

We also can take the complete file instead:
https://github.com/wix/wix-style-react/blob/master/.prettierrc.js

And then WebStorm might find the file (not sure if it detects the config when its inside the package.json)